### PR TITLE
VoLTE support

### PR DESCRIPTION
**Pull request by karthick111**

### CODE REVIEW

**Comment by karthick111** [comment]
@skritchz

**Comment by skritchz** [comment]
Yeah thanks a lot for the pr. it's the weekend so family time first. I'll
take a look when I have time or on Monday. But I seems it's all good.

**Comment by karthick111** [comment]
Take your time :)

**Review by darshan1205 — APPROVED** [review]

**Review by skritchz — COMMENTED** [review]
About the system.prop changes.

Some of those values should be put in the init_a6020.c like the lteOnCdmaDevice which is set in there and the ro.telephony.default_network, this depends on the dual/single sim device

Are you sure that all those CDMA things are needed for the VoLTE. ril subscription is also for CDMA.

**Comment by karthick111** [comment]
@skritchz  we can remove "ro.telephony.default_network=9,9" haven't noticed that it was in init_a6020.c

**Comment by Shr3ps** [comment]
Can you squash your commits? Would be better for a proper merge

File: AndroidBoard.mk [hunk 1]
```diff
@@ -4,5 +4,36 @@ include $(CLEAR_VARS)
 
 ALL_PREBUILT += $(INSTALLED_KERNEL_TARGET)
 
+#
```
Comment (Shr3ps) [inline AndroidBoard.mk:7]: This is dirty and should not be here.

File: device_A6020.mk [hunk 2]
```diff
@@ -231,7 +231,8 @@ PRODUCT_PACKAGES += \
     tcpdump \
     wcnss_service
 
-
```
Comment (Shr3ps) [inline device_A6020.mk:234]: A proper comment here would be nice.

File: overlay/frameworks/base/core/res/res/values/config.xml [hunk 3]
```diff
@@ -262,6 +261,9 @@
     <bool name="config_dozeAfterScreenOff">true</bool>
     <bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>
 
+    <!-- Whether device supports double tap to wake -->
```
Comment (Shr3ps) [inline overlay/frameworks/base/core/res/res/values/config.xml:264]: Unrelated to VoLTE. Should not be here.

File: rootdir/etc/init.qcom.rc [hunk 4]
```diff
@@ -945,3 +910,42 @@ on fs
     write /sys/class/android_usb/android0/iProduct ${ro.product.model}
     write /sys/class/android_usb/android0/iSerial ${ro.serialno}
 
+on init
```
Comment (Shr3ps) [inline rootdir/etc/init.qcom.rc:913]: Why an empty on init ?

File: proprietary-files.txt [hunk 5]
```diff
@@ -17,6 +17,7 @@ vendor/lib/libadiertac.so
 vendor/lib/libaudcal.so
 vendor/lib/libhwdaphal.so
 vendor/lib/libqct_resampler.so
+vendor/lib/libvoice-svc.so
```
Comment (Shr3ps) [inline proprietary-files.txt:20]: This list should be updated according to https://github.com/boulzordev/android_vendor_lenovo_A6020/pull/4/

**Comment by chaubeyprateek** [comment]
@Shr3ps
Squashing all in one will remove original authorship
Plus whole tree is derped by you guys

**Pull request closed without merging**